src/NServiceBus.Core/Serialization/SerializationSettingsExtensions.cs

```diff
@@ -0,0 +1,48 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Collections.Generic;
+    using Serialization;
+    using Settings;
+
+    static class SerializationSettingsExtensions
+    {
+        const string AdditionalSerializersSettingsKey = "AdditionalDeserializers";
+
+        public static void ApplySerializationDefaults(this SettingsHolder settings)
+        {
+            settings.SetDefault(AdditionalSerializersSettingsKey, new List<Tuple<SerializationDefinition, SettingsHolder>>());
+        }
+
+        public static List<Tuple<SerializationDefinition, SettingsHolder>> GetAdditionalSerializers(this SettingsHolder settings)
+        {
+            List<Tuple<SerializationDefinition, SettingsHolder>> deserializers;
+            if (!settings.TryGet(AdditionalSerializersSettingsKey, out deserializers))
+            {
+                deserializers = new List<Tuple<SerializationDefinition, SettingsHolder>>();
+                settings.Set(AdditionalSerializersSettingsKey, deserializers);
+            }
+            return deserializers;
+        }
+
+        public static List<Tuple<SerializationDefinition, SettingsHolder>> GetAdditionalSerializers(this ReadOnlySettings settings)
```

Comment:
do we need this overload? SettingsHolder implements ReadOnlySettings and therefore we should be able to use the extension above?